master/getting-started/kubernetes/installation/migration-from-flannel.md

```diff
@@ -0,0 +1,155 @@
+
+   ```
+   kubectl delete -f {{site.url}}/{{page.version}}/manifests/flannel-migration/migration-job.yaml
+   ```
+# Configuration options
+
+The migration controller will auto-detect your flannel configuration, and in most cases does not require any
+additional configuration. If you do require special configuration, the controller exposes the following options
+which can be set as environment variables within the pod.
+
+| Configuration options            | Description                                                          | Default                                    |
+|----------------------------------|----------------------------------------------------------------------|--------------------------------------------|
+| FLANNEL_NETWORK                  | IPv4 network CIDR used by flannel for the cluster.                   | <Automatically detected>                   |
+| FLANNEL_DAEMONSET_NAME           | Name of the flannel daemon set in the kube-system namespace.         | kube-flannel-ds-amd64                      |
+| FLANNEL_MTU                      | MTU for the flannel VXLAN device.                                    | <Automatically detected>                   |
+| FLANNEL_IP_MASQ                  | Whether masquerading is enabled for outbound traffic.                | true                                       |
+| FLANNEL_SUBNET_LEN               | Per-node subnet length used by flannel.                              | 24                                         |
+| FLANNEL_ANNOTATION_PREFIX        | Value provided via the kube-annotation-prefix option to flannel.     |  flannel.alpha.coreos.com                  |
+| FLANNEL_VNI                      | The VNI used for the flannel network.                                |  flannel.alpha.coreos.com                  |
+| FLANNEL_PORT                     | UDP port used for VXLAN.                                             |  8472                                      |
+| CALICO_DAEMONSET_NAME            | Name of the calico daemon set in the kube-system namespace.          |  calico-node                               |
+| CNI_CONFIG_DIR                   | Full path on the host in which to search for CNI config files.       |  calico-node                               |
+
+# Troubleshooting
+
+## Checking migration status
+
+The migration controller should run to completion. You can run the following command to view the controller's current status.
+
+```
+kubectl get pods -n kube-system -l k8s-app=flannel-migration-controller
+```
+
+## Viewing migration logs
+
+The migration controller logs can be viewed with the following command:
+
+```
+kubectl logs -n kube-system -l k8s-app=flannel-migration-controller
+```
+
+The logs will indicate if any actions need to be taken by the user.
+
+# Rollback
+
+Migration from Calico to flannel is not supported. If a problem is encountered during the migration, you can use the following steps
```

Comment:
Migration from Calico to Flannel is not supported. If you experience a problem during the migration, follow these steps: